src/datasources/accounts/accounts.datasource.ts

```diff
@@ -25,7 +26,13 @@ import postgres from 'postgres';
 
 @Injectable()
 export class AccountsDatasource implements IAccountsDatasource, OnModuleInit {
+  private static readonly SIMPLE_IPV4_REGEX = /^(\d{1,3}\.){3}\d{1,3}$/;
```

Comment:
We could use zod to validate this.

Reply:
Added in [b2e5161](https://github.com/safe-global/safe-client-gateway/pull/1800/commits/b2e51613dcbc65252b8e6f2865027a72adc25644)